test/built-ins/Object/getOwnPropertyDescriptors/symbols-included.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2016 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Object.getOwnPropertyDescriptors includes Symbol keys.
+id: pending
+author: Jordan Harband
+features: [Symbol]
+---*/
+
+var value = {};
+var enumSym = Symbol('enum');
+var nonEnumSym = Symbol('nonenum');
+var symValue = Symbol('value');
+
+var obj = { key: symValue };
+obj[enumSym] = value;
+Object.defineProperty(obj, nonEnumSym, { enumerable: false, value: value });
+
+var result = Object.getOwnPropertyDescriptors(obj);
+
+assert.sameValue(Object.keys(result).length, 3, 'obj has 3 descriptors');
```

Comment:
Object.keys() does not include Symbols, only Strings --- so this doesn't really work. See https://build.chromium.org/p/tryserver.v8/builders/v8_linux64_asan_rel/builds/14411/steps/Test262%20-%20no%20variants/logs/symbols-included/text